views/main.js

```diff
@@ -132,28 +132,35 @@ function render(data) {
     $answers.html('');
 
   // Back Button
-  if (actions.length) {
-    var $backBtn = $('<button class="back-button">Back</button>');
-    $backBtn.click(function() {
-      var previous = actions.pop();
-
-      var checklistIdx = checklist.indexOf(previous.checklist);
-      if (checklistIdx >= 0) {
-        checklist.splice(checklistIdx, 1);
-      }
-
-      render(previous);
-    });
+    if (actions.length) {
```

Comment:
This is calling the new function instead of constructing the back button inline, but it functions the same as before.